modules/claims_api/app/swagger/claims_api/v1/form_0966_controller_swagger.rb

```diff
@@ -169,16 +169,49 @@ class Form0966ControllerSwagger
               end
             end
           end
-          response :default do
-            key :description, 'unexpected error'
+
+          response 401 do
+            key :description, 'Unauthorized'
+            content 'application/json' do
+              schema do
+                key :type, :object
+                key :required, [:errors]
+                property :errors do
+                  key :type, :array
+                  items do
+                    key :'$ref', :NotAuthorizedModel
+                  end
+                end
+              end
+            end
+          end
+
+          response 403 do
+            key :description, 'Forbidden'
+            content 'application/json' do
+              schema do
+                key :type, :object
+                key :required, [:errors]
+                property :errors do
+                  key :type, :array
+                  items do
+                    key :'$ref', :ForbiddenModel
```

Comment:
This error example doesn't seem right?
![image](https://user-images.githubusercontent.com/31224301/108910523-9e633c00-75eb-11eb-8fd8-6b474e55659c.png)

Reply:
Dang!
I thought I double-checked each endpoint on both versions of the API.  😞 
Fixed it [here](https://github.com/department-of-veterans-affairs/vets-api/pull/5992/commits/8d9e11a6b894ad4caec09bafdc4a74c0541f414b).